### ISSUE TITLE
Memory leak in gRPC PHP

### ISSUE THREAD

**Issue by michaelbausor** [grpc/grpc#10996]
cc @stanley-cheung @tmatsuo @bshaffer

Code to reproduce in this gist: https://gist.github.com/michaelbausor/f62d18352d892ae1a452f8cfd63ad29e
Note that this code relies on protobuf types generated using the latest protobuf implementation, but which are not yet published. I can help to reproduce.

The memory consumption printed grows by approx 2K per call.

**Comment by stanley-cheung** [comment]
I am working on PR #11003 which should hopefully fix the memory leaks.

Before the fix, running the script linked from the top of this issue:
```
memory used: 3820592
memory used: 3822856
memory used: 3825080
memory used: 3827304
memory used: 3829528
memory used: 3831752
memory used: 3833976
memory used: 3836200
memory used: 3838424
memory used: 3840648
memory used: 3842872
memory used: 3845096
memory used: 3847320
memory used: 3849544
memory used: 3851768
memory used: 3853992
memory used: 3856216
memory used: 3858440
memory used: 3860664
memory used: 3862888
memory used: 3865112
memory used: 3867336
```


After the fix,
```
memory used: 3818472
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
memory used: 3818504
```

**Comment by stanley-cheung** [comment]
Closing this. This should be fixed in the `1.3.2` version released today.
